src/View/addons/ImageFusionAddOnOption.py

```diff
@@ -67,13 +72,19 @@ def setupUi(self):
         self.gridLayout.setContentsMargins(0, 0, 0, 0)
         self.gridLayout.setVerticalSpacing(0)
 
+        hspacer = None
         # Create horizontal spacer in the middle of the grid
         try:
             # PySide6 worked on Windows
-            hspacer = QtWidgets.QSpacerItem(QtWidgets.QSizePolicy.Expanding,QtWidgets.QSizePolicy.Minimum)
+            hspacer = QtWidgets.QSpacerItem(5,
+                                            16,
+                                            QtWidgets.QSizePolicy.Policy.Expanding,
+                                            QtWidgets.QSizePolicy.Policy.Minimum
+                                            )
         except TypeError:
             # PySide6 needed for Ubuntu 22.04 (perhaps due to PySide 6.4.0.1)
             # no idea if these are appropriate default values
+            logger.error("Type Error when defining spacer item")
             hspacer = QtWidgets.QSpacerItem(16,5) 
```

Comment:
these are magic numbers in the sense that there is no explanation of where they came from, nor are they explicit named constants.

Reply:
Agreed